src/main/scala/guessbook/GuessBookUserInput.scala

```diff
@@ -0,0 +1,206 @@
+package guessbook
+
+import guesswho.GenderEnum
+
+import scala.io.StdIn
+import scala.util.{Failure, Success, Try}
+
+object GuessBookUserInput {
+  def main(args: Array[String]): Unit = {
+    val game = GuessBook
+    println("START OF GAME\n")
+
+    // Randomly select a book for each player
```

Comment:
could you ask the user which mode they want to play it on easily (either random book, or player vs player)?